fedimint-api/src/module/mod.rs

```diff
@@ -241,16 +245,21 @@ pub trait ModuleInit {
 
 #[async_trait]
 pub trait ServerModulePlugin: Debug + Sized {
+    const KIND: ModuleKind;
+
     type Decoder: PluginDecode;
     type Input: PluginInput;
     type Output: PluginOutput;
     type OutputOutcome: PluginOutputOutcome;
     type ConsensusItem: PluginConsensusItem;
```

Comment:
Follow-up (I can tackle this): remove these types since they are in `Decoder` now

Reply:
Oh.... Smart, smart. Crossed my mind that we have a redundancy.